app/src/main/res/layout/history_fragment.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/colorPrimary"
+        tools:context=".ui.main.HistoryFragment">
+
+
+    <TextView
+        android:id="@+id/id_label"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_marginTop="@dimen/extremely_huge_margin"
+        style="@style/AppText.Header"
+        android:textColor="@color/white"
+        android:text="@string/dashboard_info_your_id" />
+
+    <TextView
+        android:id="@+id/ble_id"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/id_label"
+        style="@style/AppText.Header"
+        android:textColor="@color/white"
+        android:text="XXX XXX XXX XXX" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/back_button"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/large_margin"
+        android:backgroundTint="@color/white"
+        android:layout_marginHorizontal="@dimen/large_margin"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/ble_id"
+        style="@style/Button.Secondary"
+        android:text="@string/dashboard_info_send_data_btn"   />
+
+
+    <TextView
+        android:id="@+id/history_summary"
+        style="@style/AppText.StatusHeader"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/extremely_huge_margin"
+        android:textColor="@color/white"
+        android:gravity="center"
+        app:layout_constraintBottom_toTopOf="@id/e_mail_label"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        android:layout_marginHorizontal="@dimen/extremely_huge_margin"
+        android:text="Od godziny 13:00 pojawiło się wokół Ciebie 7 telefonów z aktywnie działającą aplikacją"/>
```

Comment:
```suggestion
        android:text="@string/dashboard_info_history_overview"/>
```

Reply:
To nie jest ten napis. Obecnie nie mamy zaimplementowanej jeszcze funkcjonalności związanej z BLE więc ta wiadomość nie jest składana i mamy jedynie placeholder

Reply:
No tak, ale to docelowo ma być tam ten identyfikator i jest on już zaimportowany do resource'ów. :)

Reply:
Nie, nie ten identyfikator. Docelowy tekst to kilka resourceów połączonych w jeden z dodatkowymi parametrami.